src/store/modules/key-value.js

```diff
@@ -0,0 +1,136 @@
+import { api, loadingDataViaLoop } from '@/api'
+
+function getKvValue (kvKey, kvArray) {
+  const kvFound = kvArray.find((key) => key.id === kvKey)
+
+  if (!kvFound) {
+    return ''
+  }
+
+  return kvFound.value.u32
+}
+
+export const state = {
+  defaultRoleIds: {
+    general: null,
+    corporate: null,
+    usVerified: null,
+    usAccredited: null,
+    unverified: null,
+    blocked: null,
+  },
+  kvAssetTypeDefault: null,
+  kvAssetTypeKycRequired: null,
+  kvAssetTypeSecurity: null,
+  kvPollTypeRestricted: null,
+  kvPollTypeUnrestricted: null,
+  kvEntries: null,
+}
+
+export const mutations = {
+  SET_KV_ENTRY_GENERAL_ROLE_ID (state, id) {
+    state.defaultRoleIds.general = id
+  },
+
+  SET_KV_ENTRY_CORPORATE_ROLE_ID (state, id) {
+    state.defaultRoleIds.corporate = id
+  },
+
+  SET_KV_ENTRY_US_VERIFIED_ROLE_ID (state, id) {
+    state.defaultRoleIds.usVerified = id
+  },
+
+  SET_KV_ENTRY_US_ACCREDITED_ROLE_ID (state, id) {
+    state.defaultRoleIds.usAccredited = id
+  },
+
+  SET_KV_ENTRY_UNVERIFIED_ROLE_ID (state, id) {
+    state.defaultRoleIds.unverified = id
+  },
+
+  SET_KV_ENTRY_BLOCKED_ROLE_ID (state, id) {
+    state.defaultRoleIds.blocked = id
+  },
+
+  SET_KV_ASSET_TYPE_DEFAULT (state, value) {
+    state.kvAssetTypeDefault = value
+  },
+
+  SET_KV_ASSET_TYPE_KYC_REQUIRED (state, kvAssetTypeKycRequired) {
+    state.kvAssetTypeKycRequired = kvAssetTypeKycRequired
+  },
+
+  SET_KV_ASSET_TYPE_SECURITY (state, kvAssetTypeSecurity) {
+    state.kvAssetTypeSecurity = kvAssetTypeSecurity
+  },
+
+  SET_KV_POLL_TYPE_RESTRICTED (state, kvPollTypeRestricted) {
+    state.kvPollTypeRestricted = kvPollTypeRestricted
+  },
+
+  SET_KV_POLL_TYPE_UNRESTRICTED (state, kvPollTypeUnrestricted) {
+    state.kvPollTypeUnrestricted = kvPollTypeUnrestricted
+  },
+
+  SET_KV_ENTRIES (state, data) {
+    state.kvEntries = data
+  },
+}
+
+export const actions = {
+  async LOAD_KV_ENTRIES ({ dispatch }) {
+    await dispatch('LOAD_KV_ENTRIES_ACCOUNT_ROLE_IDS')
+  },
+
+  async LOAD_KV_ENTRIES_ACCOUNT_ROLE_IDS ({ commit }) {
```

Comment:
the action named `LOAD_KV_ENTRIES_ACCOUNT_ROLE_IDS` but actually sets not only account role IDs. I guess moving the body to `LOAD_KV_ENTRIES` would be fine